std.ddoc

```diff
@@ -1,55 +1,13 @@
 DDOC=
-<!DOCTYPE html>
-<html lang='en-US'>
-<!--
-	Copyright (c) 1999-2010 by Digital Mars
-	All Rights Reserved Written by Walter Bright
-	http://www.digitalmars.com
-  -->
-<head>
-<meta charset="utf-8" >
-<title>$(TITLE) - D Programming Language</title>
-<link rel="stylesheet" type="text/css" href="../css/codemirror.css" />
-<link rel="stylesheet" type="text/css" href="../css/style.css">
-<link rel="stylesheet" type="text/css" href="../css/cssmenu.css">
-<meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=0.1, maximum-scale=10.0" />
-</head>
+$(DOCTYPE_OPENHTML_COPYRIGHT_HEAD)
 <body id="$(TITLE)" class='hyphenate'>
 $(SCRIPT document.body.className += ' have-javascript';)
 $(DIVID top,
-	$(divid search-box,
-		<form method="get" action="http://google.com/search">
-			<img src="/images/search-left.gif" width="11" height="22" alt=""><input id="q" name="q" placeholder="Search"><input type="image" id="search-submit" name="submit" src="/images/search-button.gif">
-			<input type="hidden" id="domains" name="domains" value="dlang.org">
-			<input type="hidden" id="sourceid" name="sourceid" value="google-search">
-			$(DIVID "search-dropdown,
-				<select id="sitesearch" name="sitesearch" size="1">
-					<option value="dlang.org">Entire D Site</option>
-					<option value="dlang.org/phobos">Library Reference</option>
-					<option value="www.digitalmars.com/d/archives">Newsgroup Archives</option>
-				</select>
-			)
-		</form>
-	)
-	$(DIVID header,
-		$(A http://dlang.org, <img id="logo" width="125" height="95" alt="D Logo" src="../images/dlogo.png">)
-		<a id="d-language" href="http://dlang.org">D Programming Language</a>
-	)
-)
-$(DIVID mobile-top,
-    $(DIVID mobile-header,
-        $(DIVID mobile-hamburger, &#9776;)
-        $(DIVID mobile-title,
-            <img id='mobile-logo' width="50" height="38" alt="D Logo" src="http://dlang.org/images/dlogo.png">
-            $(DIVID mobile-d-language,
-                <a href="http://dlang.org">D$(NBSP)Programming Language</a>
-            )
-        )
-    )
-)
-$(DIVID navigation, $(TOP)
-$(NAVIGATION_PHOBOS)
+    $(TOP_SEARCHBOX ,selected,)
```

Comment:
This isn't particularly beautiful, but I couldn't think of a nicer way to pre-select an &lt;option&gt;.